package/Dockerfile.agent

```diff
@@ -2,15 +2,16 @@ ARG RANCHER_TAG=dev
 ARG RANCHER_REPO=rancher
 FROM ${RANCHER_REPO}/rancher:${RANCHER_TAG} as rancher
 
-FROM registry.suse.com/bci/bci-base:15.4
 ARG ARCH=amd64
 
-ENV KUBECTL_VERSION v1.23.6
-RUN zypper -n install --no-recommends curl ca-certificates jq git-core hostname iproute2 vim-small less \
-	bash-completion bind-utils acl openssh-clients tar gzip xz gawk sysstat && \
-    zypper -n clean -a && rm -rf /tmp/* /var/tmp/* /usr/share/doc/packages/* && \
-    curl -sLf https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/${ARCH}/kubectl > /usr/bin/kubectl && \
-    chmod +x /usr/bin/kubectl
+ARG RANCHER_BASE_TAG=base
+FROM rancher/rancher:base
```

Comment:
I know this Dockerfile is using this same base image in 2.7.5, but we should change this to use Oracle Linux as a base image at some point. Maybe create a ticket for that?

Reply:
I will create a ticket for that, great catch.